BlazarUI/app/stylus/components/commits.styl

```diff
@@ -2,62 +2,62 @@
 
 .commits
   width auto
-  border-bottom 1px solid $gray-light
-  margin-bottom 10px
+  border-bottom 1px solid $color-koala
+  margin-bottom 36px
```

Comment:
This makes the `show more` link spaced way too far apart from the commits table on the branch build details page.